src/contractWrappers/KlerosWrapper.js

```diff
@@ -447,6 +447,26 @@ class KlerosWrapper extends ContractWrapper {
     return currentRuling.toNumber()
   }
 
+  getPeriod = async (
+    contractAddress
+  ) => {
+    const contractInstance = await this.load(contractAddress)
+
+    const currentPeriod = await contractInstance.period()
+
+    return currentPeriod.toNumber()
+  }
+
```

Comment:
maybe can you use a functional syntax ? maybe like this `getPeriod = async contractAddress => await this.load(contractAddress) => async contractInstance => await contractInstance.period() => async currentPeriod => await contractInstance.period() => currentPeriod.toNumber()`